src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -517,7 +517,7 @@ private boolean checkIfBreakBuild(ScanRequest request, ScanResults results){
                 log.info("Fail build on Thresholds exceeded.");
                 breakBuildResult = true;
             }
-        } else if(flowProperties.isBreakBuild() && resultsService.filteredSastIssuesPresent(results)){
+        } else if(flowProperties.isBreakBuild() && results != null && resultsService.filteredSastIssuesPresent(results)){
```

Comment:
It's better to put the 'results != null' check inside the filteredSastIssuesPresent method.

Reply:
moved to ResultsService